### PR TITLE
Implement footer section

### PR DESCRIPTION
**Pull request by JennyDoJD**
**Design**
Mobile
image
Desktop
image

**Result**
Mobile
Screenshot (305)
Tablet
Screenshot (306)
Desktop
image

### CODE REVIEW

File: practice/src/index.html [hunk 1]
```diff
@@ -142,5 +142,67 @@ <h4 class="secondary-title">Popular Trip Packages</h4>
       </div>
     </section>
   </main>
+
+  <footer class="footer-section">
+    <!-- Footer section -->
+    <div class="footer-wrapper">
```
Comment (ghost) [inline practice/src/index.html]: @JennyDoJD could you help double the layout? It seems some div tags are unnecessary

File: practice/src/css/layouts/footer.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  gap: 4px;
+}
+
+.footer-copyright {
+  color: var(--pale-gray-text-color);
+  width: 154px;
+  padding-top: 12px;
+}
+
+.footer-col {
```
Comment (huyencao-agilityio) [inline practice/src/css/layouts/footer.css]: naming convention

File: practice/src/index.html [hunk 3]
```diff
@@ -142,5 +142,65 @@ <h4 class="secondary-title">Popular Trip Packages</h4>
...
+  <footer class="footer-section">
+    <!-- Footer section -->
+    <div class="container footer-main">
+      <div class="footer-row">
+        <div class="footer-col">
+          <a href="javascrip:void(0)" class="footer-logo">
+            <img src="../src/assets/images/footer/logo-footer.svg" alt="Logo large for footer">
+            <span class="footer-logo-name">DamiumTrip's</span>
+          </a>
+          <p class="text footer-copyright footer-text">Copyright © 2023. All Rights Reserved.</p>
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should use HTLM symbol https://www.w3schools.com/html/html_symbols.asp instead of  ©

**Comment by huyencao-agilityio** [comment]
@JennyDoJD Conflict

File: practice/src/index.html [hunk 4]
```diff
@@ -142,5 +142,65 @@ <h4 class="secondary-title">Popular Trip Packages</h4>
       </div>
     </section>
   </main>
+
+  <footer class="footer-section">
+    <!-- Footer section -->
+    <div class="container footer-main">
+      <div class="footer-row">
+        <div class="footer-coloumn">
```
Comment (canhpham-zb) [inline practice/src/index.html]: @JennyDoJD pls help check the typo

**Pull request merged**